internal/pkg/manifest/validate_test.go

```diff
@@ -211,17 +211,18 @@ func TestLoadBalancedWebService_validate(t *testing.T) {
 					ImageConfig: testImageConfig,
 					TaskConfig: TaskConfig{
 						Platform: PlatformArgsOrString{PlatformString: (*PlatformString)(aws.String("windows/amd64"))},
-						Storage: Storage{Volumes: map[string]*Volume{
-							"foo": {
-								EFS: EFSConfigOrBool{
-									Enabled: aws.Bool(true),
-								},
-								MountPointOpts: MountPointOpts{
-									ContainerPath: aws.String("mockPath"),
+						Storage: Storage{ReadonlyRootFS: aws.Bool(false),
```

Comment:
nit: can we move the value to the next line 🥺 it took me a bit to figure out what changed lol
```suggestion
						Storage: Storage{
							ReadonlyRootFS: aws.Bool(false),
```